src/Components/fe-otp/FeOtp.js

```diff
@@ -37,26 +48,50 @@ export class FeOtp extends LocalizeMixin(LitElement) {
       return;
     }
 
-    const loginData = serializedValue;
     this.dispatchEvent(
-      new CustomEvent('input-validation', { detail: loginData })
+      new CustomEvent('input-validation', { detail: serializedValue })
     );
+    this.checkOTP(ev);
+  }
+
+  async checkOTP(ev) {
+    this.isError = false;
+    try {
+      const { serializedValue } = ev.target;
+      await FeServices.postRequest({
+        url: '/otp',
+        data: {
+          code: serializedValue['otp-code'],
```

Comment:
you updated the name but not updated here

Reply:
Will change the name and add